frontend/src/components/common/file-input/FileInput.tsx

```diff
@@ -22,7 +22,7 @@ const FileInput = ({
         accept=".csv"
         disabled={disabled || isProcessing}
         onClick={(e) => ((e.target as HTMLInputElement).value = '')} // reset target value to allow selecting of new files
-        onChange={(e) => onFileSelected(e.target.files)}
+        onChange={(e) => onFileSelected(Array.from(e.target.files ?? []))}
```

Comment:
I'm a bit iffy about this...`e.target.files` has type `FileList | null` and we want to convert it to a type `File[]`.

Reply:
I think let's refactor `onFileSelected` to take in a `FileList` instead. I've looked through the code and `FileInput` is only used in the recipients pages.

Reply:
Good point -- I'll do that. Also, `e.target.files` may be `null` as well. I think I'll do a null check on this line instead of in the `onFileSelected` callbacks, since it doesn't really make sense for a caller to want to do anything with a `null` parameter.